connector/failoverconnector/testdata/config.yaml

```diff
@@ -1,7 +1,8 @@
 failover:
 
 failover/default:
-  priority_levels: [traces]
+  priority_levels:
+    - [traces]
```

Comment:
should this be `- traces`?

Reply:
The field is a [[][]](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1b4f0451d1df1c2d66cfdb4daa3345fa29cae133/connector/failoverconnector/config.go#L22)